nl_server/gcs.py

```diff
@@ -19,6 +19,8 @@
 
 from google.cloud import storage
 
+from tools.nl.embeddings import utils as embeddings_utils
```

Comment:
I'm slightly unsure about using tools libs in core server?  It might make sense to move the `download_model_from_gcs` here as `download_model` ?

In this PR, maybe duplicate function if needed, and we can dedupe in follow up.

Reply:
Done. Added a TODO as well.